src/test/java/org/htwkvisu/org/pois/AirportTest.java

```diff
@@ -0,0 +1,53 @@
+package org.htwkvisu.org.pois;
+
+import javafx.geometry.Point2D;
+import org.htwkvisu.gui.MapCanvas;
+import org.htwkvisu.scoring.ScoringCalculator;
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class AirportTest {
+
+    private static final String NAME = "Test";
+    private static final Point2D POSITION = new Point2D(0.123, -1.234);
+    private Airport airport;
+    private static final double TINY_DELTA = 0.0001;
+
+    @Before
+    public void setUp() throws Exception {
+        airport = new Airport(NAME, POSITION);
+    }
+
+    @Test
+    public void getCategoryFallOfs() throws Exception {
+        airport.getCategoryFallOfs().get(ScoringCalculator.CATEGORY_INFRASTRUCTURE);
+    }
+
+    @Test
+    public void getCoordinates() throws Exception {
```

Comment:
Please do not test getters and setters. Even the test-coverage gets higher with this tests, its not necessary to test getters and setters. You only need to test this, when you doesnt use the default implementation of the getter or setter.